core/src/main/java/com/linecorp/armeria/common/stream/FilteredStreamMessage.java

```diff
@@ -298,17 +219,21 @@ public void onNext(T o) {
             try {
                 filtered = filter(o);
             } catch (Throwable ex) {
-                StreamMessageUtil.closeOrAbort(o);
```

Comment:
We don't need to call `StreamMessageUtil.closeOrAbort(filtered);`?

Reply:
Isn't `filtered` null when we reach here?

Reply:
For context, I found a case where `o` was double-released when a maximum length was exceeded. Calling `StreamMessageUtil.closeOrAbort(o)` didn't make sense since the ownership has been transferred to `filter()` method.

Reply:
That is correct. 👍  The previous logic was wrong.